resources/assets/less/bem/beatmapset-header.less

```diff
@@ -69,6 +69,19 @@
     min-height: @header-height;
   }
 
+  &__availability-info {
+    font-size: smaller;
```

Comment:
this one as well.... or not. Not even mdn can tell how much smaller is `smaller` though.